### DISCUSSION TITLE
Enerygtotal unplanned reset to 0 on Tasmota 10.1.0

### DISCUSSION THREAD

**Discussion by funky4t** [arendst/Tasmota#14888] [Q&A] [answered]
Hi everyone,
I have a strange behaviour on a Sonoff with Tasmota 10.1.0.
I used two of them for measuring energy of washing machine and tumble dryer, exact same config, as far as I can tell. The thing is, that my dryer reseted totalenergy on 14/02 back to zero. I can remember I had a short power outtake, although I cannot say ift that was on 14/02, but it is the only external factor I can think of to explain why the dryer Sonoff resets energytotal at all. Curious thing is that the washing Sonoff also got the power outtake but restarted without energytotal reset and the old values.

that is the influxdb history of the dryer Sonoff
![grafik](https://user-images.githubusercontent.com/57875762/154450883-24aa8860-3c1b-4270-9dfb-90421f24d56f.png)

So in case I am right and the powerouttake was the general trigger for that behaviour, I guess there must be a different setoption in the dryer Sonoff that resets the energytotal to zero after powerout while the washing Sonoff restarts with the old values. Another ideas is, that the dryer Sonoff has a "time or value-based" trigger that triggers instead of the power outtake. But I can´t remember to have set that, especially as I did not even know where to set that :( I searched the commands but I didn´t really found the right command/option to check inside the dryer Sonoff. Is there anyone who can point me in a more precise direction? Thanks in advance for your help :) Final goal is, that the dryer Sonoff keeps the totalenergy after powerouttake, as the washing Sonoff already does... :)

**Comment by sfromis** [comment, marked as answer]
A power outage might not be a clean cut, but unstable/brownouts, which has the risk of triggering "fast power cycle recovery" or boot loop detection, with details of such detection possibly depending on minute hardware differences, normally inside the scope of normal.

SetOption65 can be a consideration to disable the recovery feature, of the power input can behave badly.

**Reply by funky4t** [reply]
> Thanks for that idea, looks like it is a point to start with. Washing machine was set to "0" and dryer to "1". I "produced" the clean blackout for a minute myself, but I am not sure aboute the date...  But I really can remeber one slight brownout to the last days, so yes option65 could cause the reset....although I always thougt that will reset the whole device and not only energytotal.... Anyway, I changed it to "0" for the dryer an will observe it the next couple of days... Thank YOU so far
